### PR TITLE
Fix skip_inference option in python

### PR DESCRIPTION
**Pull request by ir5**
#508 caused a bug: computation_order option will not work from python code.
This PR will fix the issue.

### CODE REVIEW

File: chainer_compiler/chainer_compiler.py [hunk 1]
```diff
@@ -238,10 +238,8 @@ def __init__(self, model, onnx_file, used_translator, dump_onnx=False,
 
     def compile(self, onnx_file):
         # TODO(hamaji): Revive shape inference.
```
Comment (shinh) [inline chainer_compiler/chainer_compiler.py]: Could you move this comment to line 262 again?

File: chainer_compiler/chainer_compiler.py [hunk 2]
```diff
@@ -262,6 +260,8 @@ def compile(self, onnx_file):
                              bwd_graph.dump() +
                              '\n=== ^^^ backward ^^^ ===\n')
 
+        _chainer_compiler_core.configure(skip_inference=True)
```
Comment (shinh) [inline chainer_compiler/chainer_compiler.py]: I think the original logic,
```
        compiler_kwargs = {'skip_inference': True}
        if self.compiler_kwargs is not None:
            compiler_kwargs.update(self.compiler_kwargs)
        _chainer_compiler_core.configure(**compiler_kwargs)
```
is more appropriate here. Doesn't this work for you?

File: chainer_compiler/chainer_compiler.py [hunk 3]
```diff
@@ -311,7 +311,7 @@ def forward(self, *args):
 
         runtime_kwargs = {}
         if (self.runtime_kwargs is not None and
-            self.num_iterations % (self.quiet_period + 1) == 0):
+           self.num_iterations % (self.quiet_period + 1) == 0):
```
Comment (shinh) [inline chainer_compiler/chainer_compiler.py]: Could you revert this unnecessary change?

File: chainer_compiler/chainer_compiler.py (hunk 2 above)
Comment (ir5) [reply chainer_compiler/chainer_compiler.py]: Because `graph.backward_to_with_order` depends on some global variables, `_chainer_compiler_core.configure` should be called before `graph.backward_to_with_order`.

File: chainer_compiler/chainer_compiler.py (hunk 1 above)
Comment (ir5) [reply chainer_compiler/chainer_compiler.py]: fixed.

File: chainer_compiler/chainer_compiler.py (hunk 3 above)
Comment (ir5) [reply chainer_compiler/chainer_compiler.py]: It seems that `flake8` outputs an error at this part. But if we don't need to care, then I will revert.

File: chainer_compiler/chainer_compiler.py (hunk 2 above)
Comment (shinh) [reply chainer_compiler/chainer_compiler.py]: Yeah, I'm OK with line 242, but I think it's better to put the 4 lines here again. Otherwise, `compile` calls in line 270 and 271 will ignore `compiler_kwargs` specified by the user.

Comment (ir5) [reply chainer_compiler/chainer_compiler.py]: Ah, I understood. Thank you for pointing out this part. Now I fixed.

**Comment by shinh** [comment]
Thanks!!
